internal/peer/node/upgrade_dbs_test.go

```diff
@@ -21,5 +21,5 @@ func TestUpgradeDBsCmd(t *testing.T) {
 	defer os.RemoveAll(testPath)
 
 	cmd := upgradeDBsCmd()
-	require.NoError(t, cmd.Execute())
+	require.EqualError(t, cmd.Execute(), "the data format is already up to date. No upgrade is required")
```

Comment:
The test name TestUpgradeDBsCmd() implies a positive test, but it is actually a negative test to verify an error, should we update the test name to indicate the scenario?